src/NuGet.Configuration/PackageSource/PackageSourceProvider.cs

```diff
@@ -84,7 +83,7 @@ private IEnumerable<PackageSource> LoadConfigurationDefaultSources(IEnumerable<P
         {
 #if !DNXCORE50
             // Global default NuGet source doesn't make sense on Mono
-            if (_isMono.Value)
+            if (RuntimeEnvironmentHelper.IsMono)
```

Comment:
This was an existing check for mono, it's not related to the casing check. I just moved the helper method.